vhdbuilder/packer/init-variables.sh

```diff
@@ -352,6 +352,8 @@ if [ "$OS_TYPE" == "Windows" ]; then
   if [ "null" != "${OS_DISK_SIZE}" ]; then
     echo "Setting os_disk_size_gb to the value in windows-settings.json for ${WINDOWS_SKU}: ${OS_DISK_SIZE}"
     os_disk_size_gb=${OS_DISK_SIZE}
+  else
+    os_disk_size_gb="30"
```

Comment:
does this impact any Windows2019 vhd size AKS built in past months?

Reply:
It does likely impact the most recent patch tuesday release. I will check the size of that VHD.